ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -510,19 +508,37 @@ public class BottomSheetController: UIViewController {
 
         var targetAlpha: CGFloat = 0.0
         if isExpandable {
+            // Offset marks top of the sheet in UIView coord space, so counterintuitively,
+            // lowest -> highest means .expanded -> .hidden
+            //             self.view top
+            // ---------------------------------------- <--- low offset
+            // ----------------------------------------
+            // -----------fully dimmed (1.0)-----------
+            // ----------------------------------------
+            // ********highest dimmed offset***********
+            // ----------------------------------------
+            // ----------------------------------------
+            // -------transition space (1.0-0.0)-------
+            // ----------------------------------------
+            // ----------------------------------------
+            // ********lowest undimmed offset**********
+            // ----------------------------------------
+            // -------- fully undimmed (0.0)-----------
+            // ----------------------------------------
+            // ---------------------------------------- <--- high offset
+            
             let currentOffset = currentSheetVerticalOffset
-            let collapsedOffset = offset(for: .collapsed)
-            let expandedOffset = offset(for: .expanded)
+            let highestDimmedOffset = offset(for: .expanded)
+            let lowestUndimmedOffset = isHeightRestricted ? offset(for: .hidden) : offset(for: .collapsed)
```

Comment:
Would it be better to make the alpha independent of the offset when the height is restricted?

Reply:
This is in preparation for my next PR, which will allow swiping further down to go to hidden state. So we still need to accommodate interactive transition of alpha between .expanded and .hidden.